### PR TITLE
Update to mockito 3

### PR DESCRIPTION
**Pull request by lfarrell**

### CODE REVIEW

File: model-fcrepo/src/test/java/edu/unc/lib/boxc/model/fcrepo/objects/FolderObjectTest.java [hunk 1]
```diff
@@ -116,13 +116,13 @@ public void addFolderTest() {
...
         WorkObjectImpl childObj = new WorkObjectImpl(childPid, driver, repoObjFactory);
-        when(repoObjFactory.createWorkObject(any(Model.class))).thenReturn(childObj);
+        when(repoObjFactory.createWorkObject(isNull())).thenReturn(childObj);
 
         WorkObject workObj = folder.addWork();
 
         verify(repoObjFactory).createWorkObject(null);
 
-        assertTrue("Incorrect type of child added", workObj instanceof WorkObject);
+        assertTrue("Incorrect type of child added", workObj != null);
```
Comment (bbpennel) [inline model-fcrepo/src/test/java/edu/unc/lib/boxc/model/fcrepo/objects/FolderObjectTest.java:125]: Why did this need to change? The factory should still be returning the WorkObjectImpl initialized above.

File: operations/src/test/java/edu/unc/lib/boxc/operations/impl/importxml/ImportXMLJobTest.java [hunk 2]
```diff
@@ -346,13 +343,7 @@ private File createTempImportFile() throws Exception {
...
 
     private void assertAddressesSet(String... expectedAddresses) throws Exception {
-        verify(msg).setFrom(argThat(new ArgumentMatcher<Address>() {
-            @Override
-            public boolean matches(Object argument) {
-                Address address = (Address) argument;
-                return address.toString().equals(FROM_EMAIL);
-            }
-        }));
+        verify(msg).setFrom(argThat(address -> address.toString().equals(FROM_EMAIL)));
```
Comment (bbpennel) [inline operations/src/test/java/edu/unc/lib/boxc/operations/impl/importxml/ImportXMLJobTest.java:346]: Nice

File: model-fcrepo/src/test/java/edu/unc/lib/boxc/model/fcrepo/objects/FolderObjectTest.java (hunk 1 above)
Comment (lfarrell) [reply model-fcrepo/src/test/java/edu/unc/lib/boxc/model/fcrepo/objects/FolderObjectTest.java:125]: I think it was suggested by Intellij. I'm fine with reverting it.

File: operations/src/test/java/edu/unc/lib/boxc/operations/impl/importxml/ImportXMLJobTest.java (hunk 2 above)
Comment (lfarrell) [reply operations/src/test/java/edu/unc/lib/boxc/operations/impl/importxml/ImportXMLJobTest.java:346]: Thank the code editor :)

**Review by bbpennel — APPROVED** [review]

**Pull request merged**